mimoto-issuers-config.json

```diff
@@ -192,6 +192,41 @@
       "qr_code_type": "OnlineSharing",
       "enabled": "true"
     },
+    {
+      "credential_issuer": "MockMdl",
+      "display": [
+        {
+          "name": "Mock Mobile Driving License",
+          "logo": {
+            "url": "https://raw.githubusercontent.com/mosip/inji-config/qa1/mosipid-logo.png",
+            "alt_text": "mosip-logo"
+          },
+          "title": "Mock Mobile Driving License",
+          "description": "Download Mock Mobile Driving License",
+          "language": "en"
+        },
+        {
+          "name": "Pilka mobiiliajokortti",
+          "logo": {
+            "url": "https://raw.githubusercontent.com/mosip/inji-config/qa1/mosipid-logo.png",
+            "alt_text": "mosip-logo"
+          },
+          "title": "Pilka mobiiliajokortti",
+          "description": "Lataa Mock Mobile -ajokortti",
+          "language": "fi"
+        }
+      ],
+      "protocol": "OpenId4VCI",
+      "client_id": "${mimoto.oidc.mdl.partner.clientid}",
+      "client_alias": "${mimoto.oidc.mdl.partner.clientid}",
+      "wellknown_endpoint": "https://${mosip.injicertify.mdl.host}/v1/certify/issuance/.well-known/openid-credential-issuer",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+      "authorization_audience": "https://${mock.identity.esignet.host}/v1/esignet/oauth/v2/token",
```

Comment:
We are using mock esignet right?

Reply:
the placeholder is pointing to esignet mock in related configMap ref